Frontend/Pages/UserManagement/ResetPassword.razor

```diff
@@ -10,38 +10,82 @@
                             </div>
                         </div>
                     </div>
                     <div class="row py-3">
                         <div class="col">
-                            <label for="email" class="form-label">Passwort bestätigen</label>
+                            <label for="confirmPassword" class="form-label">Neues Passwort bestätigen</label>
                             <div>
-                                <input type="password" id="password-repeat" class="form-control"/>@*@bind-Value=""*@
-                                @*<ValidationMessage For="() => "></ValidationMessage>*@
+                                <InputText type="password" id="confirmPassword" class="form-control" @bind-Value="_resetDto.ConfirmPassword"></InputText>
+                                <ValidationMessage For="() => _resetDto.ConfirmPassword"></ValidationMessage>
                             </div>
                         </div>
                     </div>
-                    <div class="row py-3 justify-content-center">
-                        <div class="col-6 d-grid">
-                            <button type="submit" class="btn btn-primary">Speichern</button>
+                    @if (_isSuccessful) {
+                        <div class="row py-3 justify-content-center">
+                            <p>Ihr Account wurde erfolgreich gelöscht!</p>
+                            <p>Sie werden in @_time Sekunden zur Login Seite weitergeleitet...</p>
+                        </div>
+                    } else {
+                        <div class="row py-3 justify-content-center">
+                            <div class="col-6 d-grid">
+                                <button type="submit" class="btn btn-primary">Absenden</button>
+                            </div>
                         </div>
+                    }
+                    <div class="row py-3 justify-content-center">
+                        @_errorMessage
                     </div>
-                @*</EditForm>*@
+                </EditForm>
             </div>
         </div>
     </div>
 </div>
 
 @code {
 
-}
\ No newline at end of file
+    private bool _isSuccessful;
+    private string? _errorMessage;
+
+    private int _time = 4;
+
+    [Inject]
+    HttpClient HttpClient { get; set; } = default!;
+
+    [Inject]
+    private NavigationManager NavigationManager { get; set; } = default!;
+
+    [Parameter]
+    [SupplyParameterFromQuery]
+    public string Token { get; set; } = null!;
+
+    private readonly ResetPasswordDto _resetDto = new();
+
+    private async void HandleValidSubmit() {
+        _resetDto.Token = Token;
+        var response = await HttpClient.PostAsJsonAsync("auth/reset/finish", _resetDto);
+
+        if (response.IsSuccessStatusCode) {
+            _isSuccessful = true;
+            while (_time > 0) {
+                _time--;
+                StateHasChanged();
```

Comment:
What does this loop do it looks horrible

Reply:
display seconds till redirection. i just realized still redirects if you switch to a different page before countdown has finished. thanks stackoverflow

Reply:
Why redirect at all? Just let the user navigate away themselves

Reply:
afaik most common behaviour after doing the process of resetting a password is getting redirected to login since without it you're just left on a dead page and one can assume that after resetting one's password a user would like to login

Reply:
DNC, I ain't no designer B)